frontend/src/utils/search/convertSearchParamsToProperTypes.ts

```diff
@@ -30,7 +31,18 @@ export function convertSearchParamsToProperTypes(
 }
 
 // Helper function to convert query parameters to set
-function paramToSet(param: string | string[] | undefined): Set<string> {
+function paramToSet(
+  param: string | string[] | undefined,
+  type?: string,
+): Set<string> {
+  if (type === "status") {
```

Comment:
2 (maybe 2.5) nits:
* if you put the early exit before this logic that's currently at line 46 before these conditionals I think you can save the null check on 42
* I feel slightly itchy about mutating the param argument. What about this instead?
```suggestion
  const defaultedParam = param === SEARCH_NO_STATUS_VALUE) ? "" : "forecasted,posted"
```
Or maybe this is even better since you don't have to spend time splitting the string
```suggestion
  const defaultedParam = param === SEARCH_NO_STATUS_VALUE) ? [] : ["forecasted", "posted"]
```

Reply:
Updated, thanks.